core/db/default/spree/permission_sets.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Spree::PermissionSets::Base.subclasses.each do |permission|
+  Spree::PermissionSet.create(name: permission.to_s.split('PermissionSets::').last, set: permission.to_s)
```

Comment:
LGTM as is, but wanted to mention that ActiveSupport provides some helpers to make getting class names nicer:

```
Spree::PermissionSets::Base.subclasses.each do |permission|
  Spree::PermissionSet.create!(
    name: permission.name.demodulize,
    set: permission.name
  )
end

Reply:
Fixed!